index.bs

```diff
@@ -1246,25 +1246,35 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     is not capable of [=biometric recognition=], nor does it involve the presentation of a shared secret such as a password or
     PIN.
 
+: <dfn>User Account</dfn>
+:: In the context of this specification, a [=user account=] is some section of a [=[RP]=]'s services,
+    identified by a [=user handle=]
+    and associated with some set of [=credentials=] that a user may use to gain access to that section of services.
+    The set of credentials might change over time.
```

Comment:
Hm, I'm thinking that rather than the "section of services" terminology we might try "(sub)set of resources", as in "a relying party's resources", and also mention the notion that a RP _authorizes_ access to resources on a user account basis...
```suggestion
:: In the context of this specification, a [=user account=] denotes the mapping of a set of [=credentials=] [[CREDENTIAL-MANAGEMENT-1]] to a (sub)set of a [=[RP]=]'s resources, as maintained and authorized by the [=[RP]=]. The [=[RP]=] maps a given [=public key credential=] to a [=user account=] by assigning a  [=user account=]-specific value to the credential's [=user handle=]. This mapping, the set of credentials, and their authorizations, may evolve over time.
```

Reply:
Nice, thanks!